tests/generic_vault/strategies/auraBAL/conftest.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import brownie
+from brownie import (
+    AuraBalVault,
+    AuraBalStrategy,
+    AuraBalZaps,
+    interface,
+    BBUSDHandler,
+    AuraHandler,
+)
+from ....utils.constants import (
+    AIRFORCE_SAFE,
+    AURA_BAL_TOKEN,
+    AURA_BAL_STAKING,
+    AURA_TOKEN,
+    BBUSD_TOKEN,
+    BAL_TOKEN,
+    ADDRESS_ZERO,
+)
+
+
+@pytest.fixture(scope="module")
+def vault(owner):
+    vault = AuraBalVault.deploy(AURA_BAL_TOKEN, {"from": owner})
+    vault.setPlatform(AIRFORCE_SAFE, {"from": owner})
+    yield vault
+
+
+@pytest.fixture(scope="module")
+def strategy(owner, vault):
+    strategy = AuraBalStrategy.deploy(vault, {"from": owner})
+    strategy.setApprovals({"from": owner})
+    vault.setStrategy(strategy, {"from": owner})
+    yield strategy
+
+
+@pytest.fixture(scope="module")
+def aura_handler(owner, vault, strategy):
+    handler = AuraHandler.deploy(AURA_TOKEN, strategy, {"from": owner})
+    handler.setApprovals({"from": owner})
+    yield handler
+
+
+@pytest.fixture(scope="module")
+def bbusd_handler(owner, vault, strategy):
+    handler = BBUSDHandler.deploy(BBUSD_TOKEN, strategy, {"from": owner})
+    yield handler
```

Comment:
Don't forget to add bbusd_handler.setApprovals({"from": deployer}) in case this handler needs a setApprovals as well as per my other review comment on BBUSDHandler.sol.

Reply:
![image](https://user-images.githubusercontent.com/25791237/183504760-4535cfaf-1a9a-4214-b8cc-6dbddaf9821c.png)
BPT already have max allowance for the Vault (not all of them though apparently, had to set approvals for the BAL80-WETH20 one)